docs/src/cli.md

```diff
@@ -98,7 +98,7 @@ playwright install --help
 ```
 
 ```bash csharp
-playwright install --help
+bin\Debug\netX\playwright.ps1 install --help
```

Comment:
note: netX might confuse users and it requires PowerShell which is not installed on most Linux machines. Or does .NET include PowerShell?

Reply:
Mac/Linux .net people know what to do with the powershell scripts.